lib/cdo/lesson_import_helper.rb

```diff
@@ -75,19 +75,13 @@ def self.create_lesson_resources(cb_resources, course_version_id)
   end
 
   def self.create_activity_sections(activity_markdown, lesson_activity_id, levels)
-    # Find any special syntax, such as tips or remarks, and gather them.
-    # TODO tips and remarks both show up in tip_matches. We filter out remarks
-    # but we should try to do something a bit smarter here.
+    activity_markdown.gsub!(/slide!!!slide-\d+(?:<!-- place where you'd like the icon -->)?/, "[slide]")
+    activity_markdown.gsub!(/\[\/?guide\]/, '')
     tip_matches = find_tips(activity_markdown).select {|m| m[1] != 'say'}.map {|m| {index: activity_markdown.index(m[0]), type: 'tip', match: m, substring: m[0]}}
```

Comment:
I alluded to this in the description but tips, names, pullthrough, and remarks are all syntax that _must_ be at the beginning of a line. This means that a line cannot both be a pullthrough and a remark, for instance. This allows us to look for all of these at the same time and process them all at the same time.